WcaOnRails/app/views/relations/relation.html.erb

```diff
@@ -0,0 +1,49 @@
+<% provide(:title, t('.title')) %>
+
+<div class="container text-center">
+  <h3><%= t '.find_relations' %></h3>
+
+  <%= form_tag relation_url, method: :get, class: "form-inline relations-form" do %>
```

Comment:
That's what I've done at the very beginning, but then I was really annoyed by the fact that we have a model that could be nothing else but a module with static methods... All in all it really wasn't the way I'd like to choose. That's why I made the effort of getting those wca id pickers to work "manually", it's not that bad I think ^^